e2e/suites/actions/copy.test.ts

```diff
@@ -127,8 +137,8 @@ describe('Copy content', () => {
     await apis.user.nodes.createFolder(folderSiteFav, docLibId);
     await apis.user.nodes.createFolder(folderSiteSearch, docLibId);
 
-    await apis.user.shared.waitForApi({ expect: 4 });
-    await apis.user.favorites.waitForApi({ expect: 7 });
+    await apis.user.shared.waitForApi({ expect: 5 });
+    await apis.user.favorites.waitForApi({ expect: 9 });
```

Comment:
you will have 10 favorite items, cause when you create a site it is also marked as favorite by default.
either change this expect to 10 or move it before the site creation